<launch>
	<!-- Launch this file after segbot_v2.arm.launch is launched -->
	<!-- Setting topics for the vision logging node -->
	<node name="vision_logging_server" pkg="grounded_logging" type="vision_logging_server" output="screen" cwd="node">
		<param name="rgb_topic" value="/xtion_camera/rgb/image_rect_color" />
	</node>
	
	<!-- Setting topics for the look behaviour node -->
	<node name="point_cloud_logging_server" pkg="grounded_logging" type="look_behavior" output="screen" cwd="node">
		<param name="depth_topic" value="/xtion_camera/depth_registered/points" />
	</node>
	
	<!-- Launch the audio publisher -->
	<node name="audio_sample_publisher" pkg="grounded_logging" type="audio_publisher" output="screen" cwd="node">
		<param name="audio_samples" value="/audio_samples" />
	</node>
	
	<!-- Setting topics for the audio logging node -->
	<node name="audio_logging_server" pkg="grounded_logging" type="audio_logging_server" output="screen" cwd="node">
		<param name="audio_samples" value="/audio_samples" />
	</node>
	
	<!-- Setting up topics for the haptic node -->
	<node name="arm_perceptual_log_action" pkg="segbot_arm_perception" type="log_perception_action" output="screen" cwd="node">
		<param name="tool_pos" value="/mico_arm_driver/out/tool_position" />
		<param name="finger_pos" value="/mico_arm_driver/out/finger_position" />
		<param name="joint_efforts" value="/mico_arm_driver/out/joint_efforts" />
		<param name="joint_states" value="joint_states" />
	</node>
</launch>
